.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+  pull_request:
```

Comment:
Should ideally put this on a cron to ensure integration with other modules is working correctly

You can copy the cron syntax from here https://github.com/silverstripe/silverstripe-framework/blob/4/.github/workflows/ci.yml#L8

Also include the account check so that the cron doesn't run on any forks - https://github.com/silverstripe/silverstripe-framework/blob/4/.github/workflows/ci.yml#L15

You'll also need copy paste in keepalive.yml to ensure the cron doesn't stop after 60 days of no repo activity - https://github.com/silverstripe/silverstripe-framework/blob/4/.github/workflows/keepalive.yml

Reply:
👍 

https://github.com/silverstripe/silverstripe-populate/pull/47/commits/6ff2f08a18a014b14d325b26cc352ed21338284e